### ISSUE TITLE
`TextPlacement` can't override light description

### ISSUE THREAD

**Issue by DavidGrant-NIWC** [iho-ohi/S-101-Documentation-and-FC#128]
The note below indicates that one of the intended uses of the `text` attribute is to override the light description (the other is to override the feature name):
![image](https://github.com/iho-ohi/S-101-Documentation-and-FC/assets/62906211/ed4ca666-9809-4480-8bc0-d15b53509123)

It is impossible to determine whether the light name or the light description is being overridden by the `text` attribute:
![image](https://github.com/iho-ohi/S-101-Documentation-and-FC/assets/62906211/f5e05dbb-e46b-48ee-a188-53e031cf476b)

The modeling of the `text` attribute likely pre-dates the change to the modeling of `featureName` (`nameUsage`):
* There is no reason to use `text` to override a feature name. The name can be modified on the chart via `nameUsage`.
  * Using `nameUsage` also is supports use of national language, whereas the `text` attribute doesn't.

If there is a use case for changing a light description, recommend that `text` is renamed and dedicated to that purpose.

**Comment by TomRichardson6** [comment]
I agree with Dave's point that modelling changes to feature name have overtaken this and we don't want two ways to do the same thing. Although I guess its conceivable I cant think of a really clear use case for overriding light descriptions. On this basis I would suggest we consider removing the text attribute and updating the wording accordingly given that feature name allows the alternative text to be provided on the feature to support this.

**Comment by KlasOstergren-SMA** [comment]
I agree that text should not be used to replace a feature name. Maybe this should be added to the guidance?
The only reason I can see for having a text placement with text populated that should override the light description, is to abbreviate the description. I cannot see that we will use that though.

However, I will not support the complete removal of the text attribute. I can see cases other than feature name and light description where it might be appropriate to add a text in the chart. As far as I understand the history, this is the reason why the text attribute is an alternative to the text type attribute.

For this issue, I see two options:
1: Always suppress the light description if text is populated, or
2: Prohibit the use of text to replace the light description, which is always displayed as a label in addition to the text placement text.

**Comment by DavidGrant-NIWC** [comment]
> I can see cases other than feature name and light description where it might be appropriate to add a text in the chart

What is the specific use case?

When a feature has the `featureName` attribute there's no reason to use `TextPlacement.text` to specify the text. If you want to add text and the feature doesn't bind `featureName`, then the FC should probably be updated to add the binding.

FYI, portrayal doesn't currently support `TextPlacement` for features which don't bind `featureName`:
![image](https://github.com/iho-ohi/S-101-Documentation-and-FC/assets/62906211/ffe8183e-f411-493b-9077-c8bbef15adbb)

**Comment by KlasOstergren-SMA** [comment]
> > I can see cases other than feature name and light description where it might be appropriate to add a text in the chart
> 
> What is the specific use case?
> 
It can be used in special cases where additional text clarifies the chart for the user.
For example, we have an IMO Ship Reporting System in the Sound between Sweden and Denmark. The system is divided into two sectors and when you pass from one sector to the other you have to change the VHF channel you are watching (without any reporting). This sometimes leads to potentially problematic situations due to communication issues if vessels are watching the wrong channel.  When paper charts were the main method of navigation, problems could be minimised by adding a text to the chart reminding the user to change channels. Now that more and more people are using ECDIS for navigation, the old problem returns. In this case, the ability to add an appropriate text in this case would benefit safety at sea. 

This is one use case that came to mind, but I'm sure there are others.

**Comment by DavidGrant-NIWC** [comment]
> For example, [...]

Shouldn't that use an `InformationArea` (with `featureName` providing the note on the chart)? That would support multiple languages, and also support positioning with `TextPlacement`. Additional details can also be provided via `information`.

Trying to accomplish the same thing using `TextPlacement` has many issues:
* It requires an association to a feature, and that feature must provide `featureName`.
* It doesn't support multiple languages.
* It can't provide additional information.
* It only provides point geometry, so you can't ensure it's always visible.
  * it may be past the edge of the screen.
  * The name of an `InformationArea` is shown in the visible portion(s) of the area, so it's always visible if any portion intersects the visible part of the screen.

**Comment by JeffWootton** [comment]
The discussions involving the use if `InformationArea` or `CautionArea` does not use the attribute `featureName` in order to convey the information. While `InformationArea` has `featureName` as an allowable attribute, I do not think that the intent of this is to convey information - this is the purpose of the complex attribute `information`.

The cartographic feature `TextPlacement` is a separate Agenda item for the S-101Pt13 meeting (07.8). I agree that much of the associated modelling (such as for `featureName`) has changed significantly since the `TextPlacement` feature was originally modelled. As part of the discussions on S-101PT13 Agenda 07.8, consider that the following needs to be taken into account during discussions:

- Remodelling of `featureName`.
- ECDIS performance - settings for display of names, light descriptions etc and mariner's requirements. Is `textType` required?
- Possible additional encoding guidance in teh DCEG to cover, for example, the situations that Klas has identified.
